frappe/core/doctype/user/user.json

```diff
@@ -606,7 +607,13 @@
    "fieldtype": "Link",
    "label": "Module Profile",
    "options": "Module Profile"
-  }
+  },
+  {
+    "fieldname": "reset_password_key_datetime",
+    "fieldtype": "Datetime",
+    "label": "Reset Password Key Generation Datetime",
```

Comment:
Very ambiguous. What timestamp is this? Expiry?

Reply:
This will be the timestamp when the reset_password_key was generated

Reply:
Added a description for better understancing.